charts/portal/README.md

```diff
@@ -563,12 +565,29 @@ Resulting hostnames:
 | TSSG sync | `dev-portal-sync.example.com` | `sync.example.com` | 
 | API analytics | `dev-portal-analytics.example.com` | `analytics.example.com` |
 
+## Persistent Volumes
+With the deployment of Portal, PersistentVolumeClaim (PVC) are created for components as below:
+
+- RabbitMQ - It is used by Portal containers for internal messaging. Containers publish and/or consume messages to and from RabbitMQ.
+
+Below are for Analytics:
+
+- Kafka - Kafka is responsible to stream analytics data to druid cluster. Ingestion server is the one which streams data to Kafka, which is then ingested to druid processes. If analytics containers are not available, Kafka also act as a message store and retains Analytics data upto 6hrs. 
```

Comment:
If druid containers are not available...

Reply:
Updated to Druid Containers.